input/start/declarative.md

```diff
@@ -0,0 +1,448 @@
+  <li><a href="https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl">Google Kubernetes Engine</a></li>
+  <li><a href="https://docs.openshift.com/container-platform/4.1/cli_reference/getting-started-cli.html#cli-logging-in_cli-developer-commands">Red Hat OpenShift</a> or <a href="https://docs.okd.io/latest/cli_reference/get_started_cli.html#basic-setup-and-login">OKD</a></li>
+</ul>
+
+### Set the current namespaces
+
+Use `kubectl create namespace` to create the namespaces you wish to
+use.  Use `kubectl config set-context` to set the current namespace
+for each session.
+
+<div class="code-label session-2">Console for West</div>
+
+    kubectl create namespace west
+    kubectl config set-context --current --namespace west
+
+<div class="code-label session-1">Console for East</div>
+
+    kubectl create namespace east
+    kubectl config set-context --current --namespace east
+
+
+## Step 2: Create a Skupper site in each namespace
+
+
+### Deploy the site controller
+
+<div class="code-label session-2">West</div>
+
+    $ oc apply -f https://raw.githubusercontent.com/skupperproject/skupper/0.3/cmd/site-controller/deploy-watch-current-ns.yaml
+
+<div class="code-label session-1">East</div>
+
+    $ oc apply -f https://raw.githubusercontent.com/skupperproject/skupper/0.3/cmd/site-controller/deploy-watch-current-ns.yaml
+
+**NOTE:** To configure Skupper for all namespaces in a cluster:
+
+    $ oc apply -f https://raw.githubusercontent.com/skupperproject/skupper/master/cmd/site-controller/deploy-watch-all-ns.yaml
+
+### Create a Skupper site in the east namespace
+
+<div class="code-label session-1">East</div>
+
+Create a file named `east-site.yml`
+
+    apiVersion: v1
+    data:
+      cluster-local: "false"
+      console: "true"
+      console-authentication: internal
+      console-password: "barney"
+      console-user: "rubble"
+      edge: "true"
+      name: east-site
+      router-console: "true"
+      service-controller: "true"
+      service-sync: "true"
+    kind: ConfigMap
+    metadata:
+      name: skupper-site
+
+Note that the `data.name` value of `east-site` and 
+setting `data.edge` to `true` in East disables network ingress at the
+Skupper router layer.  In our scenario, East needs to establish one
+outbound connection to West.  It does not need to accept any incoming
+connections.  As a result, no network ingress is required in East.
+
+To apply the ConfigMap:
+
+    $ oc apply -f ~/east-site.yml
+
+For more information about each parameter, see the [Site Controller README](https://github.com/skupperproject/skupper/blob/master/cmd/site-controller/README.md).
+
+After completion, you should see deployments named `skupper-service-controller` and `skupper-router`.
+
+### Create a Skupper site in the west Project
+
+<div class="code-label session-2">West</div>
+
+Create a file named `west-site.yml`
+
+    apiVersion: v1
+    data:
+      cluster-local: "false"
```

Comment:
clusterLocal?  Maybe this is a site controller question

Reply:
created https://github.com/skupperproject/skupper/issues/193